kythe/cxx/indexer/cxx/KytheGraphObserver.cc

```diff
@@ -756,6 +756,14 @@ void KytheGraphObserver::recordChildOfEdge(const NodeId& child_id,
                      VNameRefFromNodeId(parent_id));
 }
 
+void KytheGraphObserver::recordChildOfEdge(const Range& source_range,
+                                           const NodeId& parent_id) {
+  RecordAnchor(source_range, parent_id, EdgeKindID::kChildOf,
+               Claimability::Claimable);
+  // TODO(justbuchanan): stamped?
+  // TODO(justbuchanan): claimable?
```

Comment:
I'm not clear on when things should be stamped and/or claimable.

Reply:
stamped is for template instantiations, IIRC. Can't remember on Claimable, though.

Reply:
`Claimable` seems to match similar invocations, so I'll leave this as-is.